ComposureWatch/src/Components/PlayerCards.jsx

```diff
@@ -1,6 +1,160 @@
 const PlayerCards = (props) => {
-  if (props.name == "moira") {
-    return (cardImg = <img src="src/IMG/portrait-moira.png" />);
+  const charList = [
+    {
+      name: "Moira",
+      img: "src/IMG/portrait-moira.png",
```

Comment:
It might clean things up if you put these into their own javascript file and import it. Maybe something like:

```
// in this file
import charList from "../characters";

// new file src/characters.js
// notice I change the directory to images cause I like that better than IMG 😄 
import moiraPortraitUrl from "./images/portrait-moira.png";

export default [
  {
    name: "Moira",
    img: moiraPortraitUrl,
  },
  // etc..
];
```